src/org/usfirst/frc4904/autonly/SideSwitchDistance.java

```diff
@@ -1,20 +1,18 @@
 package org.usfirst.frc4904.autonly;
 
 import org.usfirst.frc4904.robot.RobotMap;
-import org.usfirst.frc4904.standard.commands.Noop;
-import org.usfirst.frc4904.standard.commands.RunIf;
 import org.usfirst.frc4904.standard.commands.chassis.ChassisMoveDistance;
-import edu.wpi.first.wpilibj.DriverStation.Alliance;
-import edu.wpi.first.wpilibj.command.CommandGroup;
 
 /**
  *
  */
-public class SideSwitchDistance extends CommandGroup {
-	public static final double DISTANCE_APPROACH_SWITCH = 9; // TODO: Test this ditance.
+public class SideSwitchDistance extends Strategy {
+	public static final double DISTANCE_APPROACH_SWITCH = 9; // TODO: Test this distance.
+	
+	public SideSwitchDistance() {
+		addSequential(new ChassisMoveDistance(RobotMap.Component.chassis, RobotMap.Metrics.Wheel.TICKS_PER_INCH * DISTANCE_APPROACH_SWITCH, RobotMap.Component.drivePID));
```

Comment:
Add a conditional to select the SideSwitchDistance-based strategies that implement Left vs. Right functionality.